conf/igenomes.config

```diff
@@ -270,7 +270,7 @@ params {
       bed12       = "${params.igenomes_base}/Homo_sapiens/UCSC/hg38/Annotation/Genes/genes.bed"
       mito_name   = "chrM"
       macs_gsize  = "2.7e9"
```

Comment:
The whole file can probably be removed as well

Reply:
Agreed - removed